html/semantics/forms/autofocus/no-cross-origin-autofocus.sub.html

```diff
@@ -36,7 +37,7 @@ <h1>Autofocus shouldn't work in cross-origin iframe.</h1>
         }
       }));
       document.getElementById("child").src =
-          "http://{{domains[www]}}:{{ports[http][0]}}/html/semantics/forms/autofocus/resources/child-autofocus.html";
+          get_host_info().HTTP_REMOTE_ORIGIN + "/html/semantics/forms/autofocus/resources/child-autofocus.html";
```

Comment:
We can remove `.sub` from the file name now, I think.

Reply:
Thanks, done.